etc/hub.api.md

```diff
@@ -4,6 +4,7 @@
 
 ```ts
 
+import type { AbortSignal as AbortSignal_2 } from 'abort-controller';

```

Comment:
why is it doing this?

Reply:
i'm guessing you didn't rebuild this file with your older abort work

Reply:
I did not, but it seems strange that it is importing as AbortSignal_2? It does the same thing with ThreadId

Reply:
maybe it just doesn't understand the `import type` system well. but related, should AbortSignal be exported to end users actually?

Reply:
We could export it yeh, its really just a import away, but easy enough to include it. Thing is, browsers already have that API, which is why we didn't include it by default. But yeh, can't hurt.